wagtailio/project_styleguide/templates/patterns/pages/feature_index_page/feature_index_page.html

```diff
@@ -0,0 +1,10 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block content %}
+    <h3>{{ page.subheading }}</h3>
+    {% for category in page.feature_categories %} ☐ {{ category }} {% endfor %}
```

Comment:
I feel like this should be handled from the block itself, the logic allows calls to the child blocks so should be dependent on that block. What do you think?

Reply:
![Screenshot from 2022-08-18 19-25-22](https://user-images.githubusercontent.com/7713776/185457543-3e4a7f21-2bb1-4156-a1cc-b98a655745d7.png)

The idea behind this line was to be able to list all the headings for the available feature categories, based on the Figma design in the image above (with a view to using JavaScript [via corresponding FE ticket] to perform the filtering). Is there a way this can be achieved on the block itself?